### DISCUSSION TITLE
Whats the meaning of Absoluteoffet field in QUIC_STREAM_EVENT_TYPE RECEIVE?

### DISCUSSION THREAD

**Discussion by shashenli** [microsoft/msquic#3075] [Q&A] [answered]
i dont know how it works

**Comment by nibanks** [comment, marked as answer]
It's the offset of the current receive, from the very beginning of the stream. For instance, you may get the following set of notifications:

- Stream start
- Stream receive (absOff=0, length=1000)
- Stream receive (absOff=1000, length=200)
- Stream receive (absOff=1200, length=1500)
- Stream receive (absOff=2700, length=800)

I hope that explains it. You're not required to make use of it, but it can help in some scenarios.

**Reply by shashenli** [reply]
thanks
